dao/src/main/java/com/anna/builder/BuilderMapSqlParameterSource.java

```diff
@@ -0,0 +1,27 @@
+package com.anna.builder;
+
+import org.springframework.jdbc.core.namedparam.MapSqlParameterSource;
+import org.springframework.stereotype.Component;
+
+@Component
+public class BuilderMapSqlParameterSource {
+
+  private static String MIN_PARAM = "minParam";
+  private static String MAX_PARAM = "maxParam";
+  private static String PAGE = "page";
+  private static String SIZE = "size";
+
+  /**
+   *Builder mapSqlParameterSource.
+   */
```

Comment:
I believe that class is not tested.
All public methods should be tested with unit tests.